### PR TITLE
feat: added cards

### PR DESCRIPTION
**Pull request by Developers-codz**

### CODE REVIEW

File: css/style/card.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    border-top-right-radius: 10px;
+}
+
+.card-img-overlay:hover {
+    opacity:0.2;
+    /* filter: grayscale(100); */
+}
+.card-img-overlay:hover + .overlay-card-content{
+   
+    display:block;
```
Comment (shahbazibnafzal) [inline css/style/card.css:62]: keep consistent spacing, else everything looks good to me

File: css/components.html [hunk 2]
```diff
@@ -6,7 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="./main.css">
-    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <!-- <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/> -->
```
Comment (shahbazibnafzal) [inline css/components.html]: remove the commented code

File: css/style/card.css (hunk 1 above)
Comment (Developers-codz) [reply css/style/card.css:62]: yeah, Tha's by mistake correcting it. Thank you for reviewing :)

File: css/components.html (hunk 2 above)
Comment (Developers-codz) [reply css/components.html]: Done!

**Review by shahbazibnafzal — APPROVED** [review]
Looks good

File: css/style/card.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
+.card{
+   position: relative;
+    background-color: var( --bg-secondary);
+    border-radius: 10px;
+    display:inline-block;
+}
+.simple-card{
```
Comment (hsnice16) [inline css/style/card.css:7]: Give at least a line space between two declarations. Else looks awesome. Nice work.

Just add a line space.

**Review by hsnice16 — APPROVED** [review]

File: css/style/card.css (hunk 3 above)
Comment (Developers-codz) [reply css/style/card.css:7]: Okay ! Thanks for reviewing :)

**Pull request merged**